ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -115,6 +115,61 @@ export async function getEditorScreenshot(
 ) {
   return await page.locator('[class*="App-module_canvas"]').screenshot(options);
 }
+const DECIMAL_PLACES = 1000;
+
+function roundToThirdDecimal(num: number) {
+  return Math.round(num * DECIMAL_PLACES) / DECIMAL_PLACES;
+}
+export async function compareFiles(file1Paths: string[], file2Paths: string[]) {
+  try {
+    const [file1Content, file2Content] = await Promise.all([
+      readFileContents(file1Paths[0]),
+      readFileContents(file2Paths[0]),
+    ]);
+
+    const file1Lines = file1Content.split('\n');
+    const file2Lines = file2Content.split('\n');
+
+    // Check if the lengths of file1Lines and file2Lines differ
+    if (file1Lines.length !== file2Lines.length) {
+      return {
+        areIdentical: false,
+        file1Content,
+        file2Content,
+      };
+    }
+
+    for (let i = 0; i < Math.min(file1Lines.length, file2Lines.length); i++) {
+      const num1 = parseFloat(file1Lines[i]);
```

Comment:
You are parsing the whole line. Example:
```
    6.5709   -4.1572    0.0000 N   0  3  0  0  0  4  0  0  0  0  0  0
```
So, obviously, it is not a number. You need to split it by spaces